src/qseek/exporters/velest.py

```diff
@@ -2,34 +2,300 @@
+        )
+        max_traveltime_delay = FloatPrompt.ask(
+            "Maximum difference between theoretical and observed arrival", default=2.5
+        )
         self.min_pick_semblance = min_pick_semblance
+        self.min_receivers_number = min_receivers_number
+        self.min_p_phase_confidence = min_p_phase_confidence
+        self.min_s_phase_confidence = min_s_phase_confidence
+        self.max_traveltime_delay = max_traveltime_delay
 
         outdir.mkdir()
         search = Search.load_rundir(rundir)
-        catalog = search.catalog  # noqa
+        phases = search.image_functions.get_phases()
+        for phase in phases:
+            if "P" in phase:
+                phase_p = phase
+            if "S" in phase:
+                phase_s = phase
+
+        catalog = search.catalog
+
+        # export station file
+        stations = search.stations.stations
+        station_file = outdir / "stations_velest.sta"
+        self.export_station(stations=stations, filename=station_file)
+
+        # export phase file
+        phase_file = outdir / "phase_velest.pha"
+        n_earthquakes = 0
+        for event in catalog:
+            if event.semblance < min_pick_semblance:
+                continue
+            if event.receivers.n_observations(phase_p) < min_receivers_number:
+                continue
+
+            observed_arrivals: list[tuple[Receiver, PhaseDetection]] = []
+
+            for receiver in event.receivers:
+                for _phase, detection in receiver.phase_arrivals.items():
+                    if detection.observed is None:
+                        continue
+                    observed = detection.observed
+                    if (
+                        detection.phase == phase_p
+                        and observed.detection_value <= min_p_phase_confidence
+                    ):
+                        continue
+                    if (
+                        detection.phase == phase_s
+                        and observed.detection_value <= min_s_phase_confidence
+                    ):
+                        continue
+                    if (
+                        detection.traveltime_delay.total_seconds()
+                        > max_traveltime_delay
+                    ):
+                        continue
+                    observed_arrivals.append((receiver, detection))
+
+            countp, counts = self.export_phases_slim(
+                phase_file, event, observed_arrivals
+            )
+            self.n_picks_p += countp
+            self.n_picks_s += counts
+            n_earthquakes += 1
+        self.n_events = n_earthquakes
+
+        # export control file
+        control_file = outdir / "velest.cmn"
+        control_file_parameters = VelestControlFile(
+            ref_lat=search.octree.location.lat,
+            ref_lon=-search.octree.location.lon,
+            n_earthquakes=n_earthquakes,
+            max_distance_station=200,
```

Comment:
Make these default values for `VelestControlFile`. Then `VelestControlFile` has to be made a `@dataclass`, which allows for default values.